apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1616,24 +1622,54 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                     </Hint>
                 </Grid.Column>
             </Grid.Row>
-            { /* TODO  Enable this option in future*/ }
-            { /*<Grid.Row columns={ 1 }>*/ }
-            { /*    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>*/ }
-            { /*        <Field*/ }
-            { /*            name="applicationAccessTokenExpiryInSeconds"*/ }
-            { /*            label="Application access token expiry time"*/ }
-            { /*            required={ true }*/ }
-            { /*            requiredErrorMessage="Please fill the application access token expiry time"*/ }
-            { /*            value={ initialValues.accessToken ?*/ }
-            { /*                initialValues.accessToken.
-                        applicationAccessTokenExpiryInSeconds.toString() :*/ }
-            { /*                metadata.defaultApplicationAccessTokenExpiryTime }*/ }
-            { /*            placeholder="Enter the application access token expiry time "*/ }
-            { /*            type="number"*/ }
-            { /*        />*/ }
-            { /*        <Hint>Configure the application access token expiry time (in seconds).</Hint>*/ }
-            { /*    </Grid.Column>*/ }
-            { /*</Grid.Row>*/ }
+
+            { /* Application AccessToken Expiry*/ }
+            { selectedGrantTypes?.includes("client_credentials") &&
+                (
+                    <>
+                        <Grid.Row columns={1}>
```

Comment:
Shall we add a space before and after the curly brace?

Reply:
Fixed with e7588c16250df30bc1610d845322a9bbfa7c322e